plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -72,10 +72,9 @@ def custom_resource_definitions
     end
 
     # Temporary template we run validations on ... so can be cheap / not fully fleshed out
-    # and only be the primary since services/configmaps are not very interesting anyway
     def verification_template
       primary_config = raw_template.detect { |e| Kubernetes::RoleConfigFile.primary?(e) } || raw_template.first
-      Kubernetes::TemplateFiller.new(self, primary_config, index: 0)
+      Kubernetes::TemplateFiller.new(self, primary_config, index: index)
```

Comment:
where is index coming from ? ... need to pass that in ?
... also please add a new test (unless is breaks the old tests)
can pair if this is getting tricky